tests/lava/test_utils/test_float2fixed.py

```diff
@@ -0,0 +1,1553 @@
+    inport: PyInPort = LavaPyType(PyInPort.VEC_DENSE, np.int16,
+                                  precision="u:16:0")
+    outport: PyOutPort = LavaPyType(PyOutPort.VEC_DENSE, np.int16,
+                                    precision="u:15:0")
+
+
+@implements(proc=ProcDense, protocol=LoihiProtocol)
+@requires(CPU)
+@tag('fixed_pt_5')
+class ProcDensePyProcModel5(PyLoihiProcessModel):
+    """A minimal fixed-point PyProcModel implementing ProcDense."""
+
+    w: np.ndarray = LavaPyType(np.ndarray, np.int32, precision="s:16:0",
+                               constant=True, scale_domain=0)
+    inport: PyInPort = LavaPyType(PyInPort.VEC_DENSE, np.int16,
+                                  precision="u:16:0")
+    outport: PyOutPort = LavaPyType(PyOutPort.VEC_DENSE, np.int16,
+                                    precision="u:15:0")
+
+
+@implements(proc=ProcDense, protocol=LoihiProtocol)
+@requires(CPU)
+@tag('fixed_pt_6')
+class ProcDensePyProcModel6(PyLoihiProcessModel):
+    """A minimal fixed-point PyProcModel implementing ProcDense."""
+
+    w: np.ndarray = LavaPyType(np.ndarray, np.int32, precision="s:16:1",
+                               constant=True, num_bits_exp=2, exp_var="exp_w")
+    inport: PyInPort = LavaPyType(PyInPort.VEC_DENSE, np.int16,
+                                  precision="u:16:0")
+    outport: PyOutPort = LavaPyType(PyOutPort.VEC_DENSE, np.int16,
+                                    precision="u:15:0")
+
+
+@implements(proc=ProcDense, protocol=LoihiProtocol)
+@requires(CPU)
+@tag('floating_pt_1', 'floating_pt_2')
+class ProcDensePyProcModelFloat2(PyLoihiProcessModel):
+    """A minimal fixed-point PyProcModel implementing ProcDense."""
+
+    w: np.ndarray = LavaPyType(np.ndarray, float, constant=True)
+    inport: PyInPort = LavaPyType(PyInPort.VEC_DENSE, float)
+    outport: PyOutPort = LavaPyType(PyOutPort.VEC_DENSE, float)
+
+
+@implements(proc=ProcRef, protocol=LoihiProtocol)
+@requires(CPU)
+@tag('fixed_pt')
+class ProcRefPyProcModelFixed(PyLoihiProcessModel):
+    """A minimal fixed-point PyProcModel implementing ProcRef."""
+
+    refport : PyRefPort = LavaPyType(PyRefPort.VEC_DENSE, int)
+    v: np.ndarray = LavaPyType(np.ndarray, np.int32, precision="s:16:0",
+                               constant=False, scale_domain=0)
+    u: np.ndarray = LavaPyType(np.ndarray, np.int32, precision="s:16:3",
+                               constant=False, scale_domain=0)
+    w: np.ndarray = LavaPyType(np.ndarray, np.int32, precision="u:12:0",
+                               constant=True, scale_domain=1)
+
+
+@implements(proc=ProcVar, protocol=LoihiProtocol)
+@requires(CPU)
+@tag('fixed_pt')
+class ProcVarPyProcModelFixed(PyLoihiProcessModel):
+    """A minimal fixed-point PyProcModel implementing ProcVar."""
+
+    v: np.ndarray = LavaPyType(np.ndarray, np.int32, precision="s:16:3",
+                               constant=False, scale_domain=0)
+
+
+class Float2Fixed(unittest.TestCase):
```

Comment:
The class should be termed "TestFloat2Fixed"

Reply:
Addressed in 8dc7625d98f36094ce348d3af28f3cc5cbd3b089.